content/cybersecurity/concepts/cyber-attack/terms/cryptojacking/cryptojacking.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'Cryptojacking Attack'
+Description: 'Cryptojacking is a type of cyberattack where a hacker uses a target’s computing power to mine cryptocurrency without the target’s knowledge or consent.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+  - 'Computer Science'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/cyber-resilience-risk-management'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+ Once the device is infected, the attacker can use its resources to mine cryptocurrency, which can be very lucrative. The most commonly mined cryptocurrency is Monero, which is preferred by attackers because it is difficult to trace. Cryptojacking can slow down the infected device and cause it to overheat, which can damage the hardware
+
+## The Impact of Cryptojacking
+
+DDoS attacks can be separated into three categories:
+
+1. *Resource Drain* : Victims experience decreased system performance, increased electricity bills, and premature hardware degradation.
+2. *Privacy Invasion* :  Cryptojacking invades user privacy as it often involves unauthorized access to computing resources.
+3. *Reputation Damage* : Organizations can suffer reputation damage if they unwittingly host cryptojacking scripts on their websites or applications.
+
+## How Bug Bounty Hunters Can Help
+
+1. *Identify Cryptojacking Scripts →* Look for suspicious scripts or code on websites or within applications you are testing. Cryptojacking scripts often consume excessive CPU resources and can be hidden in the code.
```

Comment:
[proofreading] See [my comment in line 19](https://github.com/Codecademy/docs/pull/3466/files#r1349782435) for context. 
```suggestion
1. *Identify Cryptojacking Scripts →* Look for suspicious scripts or code on websites or within applications you are testing. Cryptojacking scripts often consume excessive [CPU](https://www.codecademy.com/resources/docs/general/computer-hardware/cpu) resources and can be hidden in the code.
```